src/standard-live-connect.js

```diff
@@ -105,15 +104,20 @@ function _processArgs (args, pixelClient, enrichedState) {
  */
 function _getInitializedLiveConnect (liveConnectConfig) {
   try {
-    if (window && window.liQ && window.liQ.ready) {
-      const mismatchedConfig = window.liQ.config && _configMatcher(window.liQ.config, liveConnectConfig)
+    if (window && window[liveConnectConfig.globalVarName] && window[liveConnectConfig.globalVarName].ready) {
+      const mismatchedConfig = window[liveConnectConfig.globalVarName].config && _configMatcher(window[liveConnectConfig.globalVarName].config, liveConnectConfig)
       if (mismatchedConfig) {
         const error = new Error()
         error.name = 'ConfigSent'
         error.message = 'Additional configuration received'
-        emitter.error('LCDuplication', JSON.stringify(mismatchedConfig), error)
+        const eventBus = getAndAttachGlobalBus(liveConnectConfig.globalVarName)
+        eventBus.emitErrorWithMessage('LCDuplication', JSON.stringify(mismatchedConfig), error)
       }
-      return window.liQ
+      return window[liveConnectConfig.globalVarName]
+    }
+    if (liveConnectConfig.globalVarName !== 'liQ') {
+      window.liQ_instances = window.liQ_instances || []
+      window.liQ_instances.push(liveConnectConfig.globalVarName)
```

Comment:
Why not adding `liQ` to the array too?

Reply:
Could we put this functionality somewhere else? It seems to add to the complexity of `_getInitializedLiveConnect`, but does not have anything to do with actually getting an initialized LC.

Reply:
We could even save references to the initializes LCs instead of their global names.

Reply:
Looks good. Left a comment.